src/components/kytos/napp/NappsInfoPanel.vue

```diff
@@ -1,101 +1,102 @@
 <template>
- <div class='k-napps-info-panel'>
-  <component v-show="false" v-for="component in components" :is="component.name" v-bind:key="component.name"></component>
- </div>
-</template>
-
-<script>
-import Vue from 'vue'
-import { loadModule } from "vue3-sfc-loader"
+  <div class='k-napps-info-panel'>
+   <component v-show="false" v-for="component in components" :is="component.name" v-bind:key="component.name"></component>
+  </div>
+ </template>
  
-const options = {
+ <script>
+ import { defineAsyncComponent } from 'vue'
+  import { loadModule } from 'https://cdn.jsdelivr.net/npm/vue3-sfc-loader@0.9.5/dist/vue3-sfc-loader.esm.js' //'vue3-sfc-loader'
```

Comment:
I think you can revert to "vue3-sfc-loader". It will be hard to track this version of the library if it is hardcoded.